### PR TITLE
Fixes #1 Added bug issue template

### LINKED ISSUE #1: No issue template
currently, there is no issue template for this repo. I think it would be great if we have one.

### PR DESCRIPTION
**Pull request by danishjamal104**
fixes #1

### CODE REVIEW

**Comment by ayush0x00** [comment]
Fix the typo in your PR message

File: .gtihub/ISSUE_TEMPLATE/bugs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+A clear and concise description of what the bug is.
+
+Area Where Bug Occurred
+- [ ] Lambda <@<mention name of lambda here if selected>>
+- [ ] DynamodDb
+
+**How to reproduce the error/bug**
+(add a step here if exist)
+
+**Additional context**
```
Comment (ayush0x00) [inline .gtihub/ISSUE_TEMPLATE/bugs.md:20]: Add hardware and software specifications

**Review by ayush0x00 — APPROVED** [review]
All good.

**Comment by ayush0x00** [comment]
The commit message of your PR has a typo..please fix that.

**Review by chhavituteja — APPROVED** [review]

**Pull request merged**